### PR TITLE
Add births to the aged-on population

### PR DESCRIPTION
**Pull request by Eleonore9**

### CODE REVIEW

File: src/witan/models/load_data.clj [hunk 1]
```diff
@@ -80,6 +80,21 @@
   {:column-names (apply-col-names-schema PopulationSchema csv-data)
    :columns (vec (apply-row-schema PopulationSchema csv-data))})
 
+(defmethod apply-rec-coercion :births
```
Comment (otfrom) [inline src/witan/models/load_data.clj:83]: why not load up the data into a dataset and then just check the dataset w/the schema?

**Comment by otfrom** [comment]
Worth thinking about that load-data ns, but otherwise :shipit:

File: src/witan/models/load_data.clj (hunk 1 above)
Comment (LoraArmstrong) [inline src/witan/models/load_data.clj:83]: Because here we're coercing, not validating. Is there an easier way to do this?

Comment (otfrom) [inline src/witan/models/load_data.clj:83]: I'm not sure if you can do that inline or not. Worth a test?

Comment (LoraArmstrong) [inline src/witan/models/load_data.clj:83]: We're coercing the data before it is a dataset (when it is still in vectors), using a coercion fn and the row schemas that you helped us make from col schemas. I'm inclined to leave this for now & talk to you about improvements in the future.

Comment (otfrom) [inline src/witan/models/load_data.clj:83]: I agree it can be left for now. Just one to think about from a design point of view.

**Pull request merged**